docs/streams/developer-guide/running-app.html

```diff
@@ -134,6 +135,22 @@ <h3><a class="toc-backref" href="#id8">Determining how many application instance
                       <li>Data should be equally distributed across topic partitions. For example, if two topic partitions each have 1 million messages, this is better than a single partition with 2 million messages and none in the other.</li>
                       <li>Processing workload should be equally distributed across topic partitions. For example, if the time to process messages varies widely, then it is better to spread the processing-intensive messages across partitions rather than storing these messages within the same partition.</li>
                   </ul>
+              </div>
+              <div class="section" id="handling-crashes-and-failures">
+                <span id="streams-developer-guide-execution-scaling"></span><h2><a class="toc-backref" href="#id9">Handling crashes and failures</a><a class="headerlink" href="#handling-crashes-and-failures" title="Permalink to this headline"></a></h2>
+                <p>There are a few things you can do to reduce the likelihood of crashes and failures of your Kafka Streams application.
+                    <ul class="simple">
+                        <li>Kafka Streams has a few configurations that can help with resilience in the face of broker failures. They can be found here: <a class="reference internal" href="config-streams.html#recommended-configuration-parameters-for-resiliency">Recommended configuration parameters for resiliency</a></li>
```

Comment:
> They can be found here

It is bad writing style to use "here". Text should be written as if it was a book, with no hyperlinks, and the sentence should still make sense w/o any hyperlinks.

So something like:
> They can be found in the configuration guide.

and just make "configuration guide" the link.